src/content/get-started/use-a-model.mdx

```diff
@@ -28,7 +28,7 @@ The examples below walk through use cases leveraging DeepSparse for testing and
   </LinkCard>
 
   <LinkCard href="./custom-use-case" heading="Custom Use Case">
-    Example for how to create a pipeline for a custom model utilizing the DeepSparse Engine.
+    Example for how to create a pipeline for a custom model utilizing DeepSparse.
```

Comment:
Since DeepSparse describes the suite of components, are you saying one can create a pipeline with the whole suite? Are all four components doing the pipeline creation?